docs/source/api.rst

```diff
@@ -7,6 +7,24 @@ API
 
 **Client**
 
+The dask distributed client is the main entry point to working with a dask cluster. It offers
+its own API for submitting and dealing with work, as well as getting information about the
+cluster it is connected to. Simply creating a client may be all that a users needs to do,
+since it will implicitly become the default engine for any dask ``compute()`` calls.
+
+There are three main ways in which ``Client`` can be called:
+- with the address of an existing scheduler, or scheduler information file; in this case,
+  the cluster has been configured elsewhere (e.g., command line, helm)
+- with a cluster instance which has been created in the current python session; in this case,
+  you have control over the cluster object (which may point to Kubernetes, for example), and
+  you may call methods on it to, for instance, scale.
+- without any connection argument; in this case, a :class:`distributed.LocalCluster` will be
```

Comment:
It will also check the `DASK_SCHEDULER_ADDRESS` env var and if it is set it will connect to that address.

Reply:
I didn't even know that.